src/OrchardCore.Modules/OrchardCore.Menu/README.md

```diff
@@ -10,7 +10,7 @@ The `Menu` shape is used to render a Menu.
 | --------- | ------------ |
 | `Model.ContentItemId` | If defined, contains the content item identifier of the menu to render. |
 | `Model.Items` | The list of menu items shapes for the menu. These are shapes of type `MenuItem`. |
-| `Model.Differentiator` | If defined, contains the formatted name of the menu. For instance `MainMenu`. |
+| `Model.Differentiator` | If defined, contains the formatted alias name of the menu. For instance `MainMenu`. |
```

Comment:
The differentiator is really the menu name

https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Menu/MenuShapes.cs#L63-L70

Reply:
I'm looking for change because I was not understanding the documentation. The code refers to menu name, and the documentation does. However when I create a menu, I only define a title and an alias. The alias defaults to being derived from the title.

The JSON has Name, DisplayName, TitlePart, AliasPart…

Is the name actually the title?

Reply:
I find code that references menu name as being contentitem displaytext.
I also found a reference of title being assigned contentitem displaytext.
So I assume title is what is referenced as name. This is what was not clear to me.

Name is not in the admin UI or the JSON.